### PR TITLE
Add default image webhook for Redis

### PR DESCRIPTION
**Pull request by abays**
I overlooked the `Redis` CRD when initially adding default image webhooks to the Infra operator.  This follows-up and adds such logic.

### CODE REVIEW

**Review by dprince — APPROVED** [review]

**Review by dciabrin — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -321,12 +321,15 @@ operator-lint: gowork ## Runs operator-lint
...
 SKIP_CERT ?=false
 .PHONY: run-with-webhook
 run-with-webhook: export METRICS_PORT?=8080
 run-with-webhook: export HEALTH_PORT?=8081
 run-with-webhook: export INFRA_CLIENT_IMAGE_URL_DEFAULT=quay.io/tripleozedcentos9/openstack-tripleoclient:current-tripleo
 run-with-webhook: export INFRA_MEMCACHED_IMAGE_URL_DEFAULT=quay.io/tripleozedcentos9/openstack-memcached:current-tripleo
+run-with-webhook: export INFRA_REDIS_IMAGE_URL_DEFAULT=registry.redhat.io/rhel9/redis-6:latest
 run-with-webhook: manifests generate fmt vet ## Run a controller from your host.
 	/bin/bash hack/configure_local_webhook.sh
 	go run ./main.go
```
[the remark this answers is not in the document]
Comment (abays) [reply Makefile]: Ack, done

[the remark this answers is not in the document]
Comment (dciabrin) [reply Makefile]: Thanks

**Review by jeckersb — APPROVED** [review]
/lgtm

**Pull request merged**